src/main/java/Deadline.java

```diff
@@ -0,0 +1,22 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected LocalDateTime by;
+
+    public Deadline(String description, LocalDateTime by) {
+        super(description);
+        this.by = by;
+    }
+
+    public LocalDateTime getBy() {
+        return by;
+    }
```

Comment:
`LocalDateTime` is an object, and when you have a getter for the object other code can access it and change it! Just a caution.